cfme/tests/storage/test_manager.py

```diff
@@ -133,3 +134,39 @@ def test_storage_manager_navigation_from_cloudprovider(manager, provider):
                                              Name=manager.name).click()
     storage_detail_view = storage_view.browser.create_view(StorageManagerDetailsView)
     assert storage_detail_view.title.text == "{} (Summary)".format(manager.name)
+
+
```

Comment:
Can you add `pytest.mark.meta(automates=[1650086])`?

Reply:
Done